Source/JavaScriptCore/b3/air/AirLowerMacros.cpp

```diff
@@ -424,7 +424,8 @@ void lowerMacros(Code& code)
 
             switch (inst.kind.opcode) {
             case ColdCCall:
-                if (code.optLevel() < 2)
+                // XXX: ARM can't currently handle ColdCCalls.
```

Comment:
WebKit does not use XXX comment.

Reply:
Ah, need to re-read the style guide. Changed to a FIXME, thanks.